pmm-templates/CPU_Utilization_Details_Cores.jsonnet

```
@@ -0,0 +1,699 @@
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local graphPanel = grafana.graphPanel;
+local annotation = grafana.annotation;
+local text = grafana.text;
+local template = grafana.template;
+local timepicker = grafana.timepicker;
+local prometheus = grafana.prometheus;
+local row = grafana.row;
+local pmmSinglestat = grafana.pmmSinglestat;
+dashboard.new(
+  'CPU Utilization Details (Cores)',
+  time_from='now-12h',
+  editable=true,
+  refresh= "1m",
+  graphTooltip='shared_crosshair',
+  schemaVersion=16,
+  version=1,
+  tags=['OS','Percona'],
+  iteration=1535381370089,
+  uid="hCWR9oimz",
+  timepicker = timepicker.new(
+    hidden = false,
+    collapse= false,
+    enable= true,
+    notice=false,
+    now= true,
+    status='Stable',
+  ),
+)
+.addAnnotation(
+  grafana.annotation.datasource(
+    'PMM Annotations',
+    '-- Grafana --',
+    enable=true,
+    hide=false,
+    type='tags',
+    builtIn=1,
+    iconColor='#e0752d',
+    limit=100,
+    tags = ['pmm_annotation'],
+  )
+)
+.addAnnotation(
+  grafana.annotation.datasource(
+    'Annotations & Alerts',
+    '-- Grafana --',
+    enable=true,
+    hide=true,
+    type='dashboard',
+    builtIn=1,
+    iconColor='#6ed0e0',
+    limit=100,
+  )
+)
+.addLink(
+  grafana.link.dashboards(
+    'Query Analytics',
+    ['QAN'],
+    type='link',
+    url='/graph/dashboard/db/_pmm-query-analytics',
+    keepTime=true,
+    includeVars=true,
+    asDropdown=false,
+    icon='dashboard',
+  )
+)
+.addLink(
+  grafana.link.dashboards(
+    'OS',
+    ['OS'],
+    keepTime=true,
+    includeVars=true,
+  )
+)
+.addLink(
+  grafana.link.dashboards(
+    'MySQL',
+    ['MySQL'],
+    keepTime=true,
+    includeVars=true,
+  )
+)
+.addLink(
+  grafana.link.dashboards(
+    'MongoDB',
+    ['MongoDB'],
+    keepTime=true,
+    includeVars=true,
+  )
+)
+.addLink(
+  grafana.link.dashboards(
+    'PostgreSQL',
+    ['PostgreSQL'],
+    keepTime=true,
+    includeVars=true,
+  )
+)
+.addLink(
+  grafana.link.dashboards(
+    'HA',
+    ['HA'],
+    keepTime=true,
+    includeVars=true,
+  )
+)
+.addLink(
+  grafana.link.dashboards(
+    'Cloud',
+    ['Cloud'],
+    keepTime=true,
+    includeVars=true,
+  )
+)
+.addLink(
+  grafana.link.dashboards(
+    'Insight',
+    ['Insight'],
+    keepTime=true,
+    includeVars=true,
+  )
+)
+.addLink(
+  grafana.link.dashboards(
+    'PMM',
+    ['PMM'],
+    keepTime=true,
+    includeVars=true,
+  )
+)
+.addTemplate(
+    template.interval(
```

Please replace this template.interval element by the variant that you use in other templates.

```suggestion
    template.interval('interval', 'auto,1s,5s,1m,5m,1h,6h,1d', 'auto', label='Interval', auto_count=200, auto_min='1s'),
```